compressor/filters/css_default.py

```diff
@@ -104,3 +104,28 @@ def url_converter(self, matchobj):
 
     def src_converter(self, matchobj):
         return self._converter(matchobj, 2, "src=%s%s%s")
+
+
+class CssRelativeFilter(CssAbsoluteFilter):
+    """
+    Do similar to ``CssAbsoluteFilter`` URL processing
+    but replace ``settings.COMPRESS_URL`` prefix with '../../'.
+    """
+
+    def add_suffix(self, url):
```

Comment:
I agree, it looks strange at glance. But I don't see a better place.

I started with the idea to override `self.input()` method. But then I realized that after `output = super().input(...)` I'll get already processed URLs so it'll be hard to detect which were _related initially_.

So I need to override some _atomic_ method that processes one URL.

There are two methods that do that:

```py
class CssAbsoluteFilter(FilterBase):
    def url_converter(self, matchobj):
        return self._converter(matchobj, 1, "url(%s%s%s)")

    def src_converter(self, matchobj):
        return self._converter(matchobj, 2, "src=%s%s%s")
    # ...
```

Method `self._converter()` does a bunch of useful things that I don't want to copy-paste, and finally, it calls `self.add_suffix()`.

`self.add_suffix()` means "add a suffix to a relative URL" which is `settings.COMPRESS_URL` in `CssAbsoluteFilter` and `'../' * (N + 1)` in `CssRelativeFilter`.

So probably it's not the worse method to override :)